src/main/java/io/appium/java_client/ios/LocksIOSDevice.java

```diff
@@ -27,11 +29,30 @@
 
```

Comment:
@mykola-mokhnach Code LGTM. Have few suggestions though,

* Can we deprecate both `LocksIOSDevice` and `LocksAndroidDevice`. Have `LocksDevice` interface instead. Currently based on this PR both android and iOS both does supports the API except this [method](https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/android/LocksAndroidDevice.java#L42) in android doesn't accept duration. May be a server fix will make the API work like iOS.

* https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/android/AndroidMobileCommandHelper.java#L182 can be removed since `MobileCommand` will have it.

* `isDeviceLocked` as like in this PR makes more sense to me instead of `isLocked` as in android now. So we  might need to deprecate that and rename in android to make it consistent.

Your thoughts?

Reply:
> Can we deprecate both LocksIOSDevice and LocksAndroidDevice. Have LocksDevice interface instead. Currently based on this PR both android and iOS both does supports the API except this method in android doesn't accept duration. May be a server fix will make the API work like iOS.

Done

> https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/android/AndroidMobileCommandHelper.java#L182 can be removed since MobileCommand will have it.

Done

> isDeviceLocked as like in this PR makes more sense to me instead of isLocked as in android now. So we might need to deprecate that and rename in android to make it consistent.

Done